tests/tracking/fluent/test_fluent_autolog.py

```diff
@@ -210,35 +214,37 @@ def log_autolog_called(self, integration, call_args, call_kwargs):
     logger = TestLogger()
     AutologgingEventLogger.set_logger(logger)
 
-    mlflow.autolog(exclusive=True, disable=True)
+    mlflow.autolog(exclusive=True, disable=False)
 
     universal_autolog_event_logging_calls = [
         call for call in logger.calls if call.integration == "mlflow"
     ]
     assert len(universal_autolog_event_logging_calls) == 1
     call = universal_autolog_event_logging_calls[0]
     assert call.integration == "mlflow"
-    assert {"disable": True, "exclusive": True}.items() <= call.call_kwargs.items()
+    assert {"disable": False, "exclusive": True}.items() <= call.call_kwargs.items()
 
 
 def test_autolog_obeys_disabled():
     from mlflow.utils.autologging_utils import AUTOLOGGING_INTEGRATIONS
+    from mlflow.utils.autologging_utils.safety import _AUTOLOGGING_PATCHES
 
     mlflow.autolog(disable=True)
     mlflow.utils.import_hooks.notify_module_loaded(sklearn)
-    assert get_autologging_config("sklearn", "disable")
+    assert "sklearn" not in _AUTOLOGGING_PATCHES
 
     mlflow.autolog()
     mlflow.utils.import_hooks.notify_module_loaded(sklearn)
     mlflow.autolog(disable=True)
     mlflow.utils.import_hooks.notify_module_loaded(sklearn)
-    assert get_autologging_config("sklearn", "disable")
+    assert "sklearn" not in _AUTOLOGGING_PATCHES
 
     mlflow.autolog(disable=False)
     mlflow.utils.import_hooks.notify_module_loaded(sklearn)
     assert not get_autologging_config("sklearn", "disable")
+    assert "sklearn" in _AUTOLOGGING_PATCHES
     mlflow.sklearn.autolog(disable=True)
-    assert get_autologging_config("sklearn", "disable")
```

Comment:
Are these changes necessary? The goal of this test is to verify that `autolog(disable=True)` propagates to framework-specific autologging integrations. It should be sufficient to check the autologging config to verify that this works properly. I don't think we need to check patch state.